src/sessions/p2p_session.rs

```diff
@@ -13,10 +13,13 @@ use std::collections::vec_deque::Drain;
 use std::collections::HashMap;
 use std::collections::VecDeque;
 use std::convert::TryInto;
+use std::hash;
 
 const RECOMMENDATION_INTERVAL: Frame = 60;
 const MIN_RECOMMENDATION: u32 = 3;
 const MAX_EVENT_QUEUE_SIZE: usize = 100;
+const CHECKSUM_REPORT_INTERVAL: Frame = 10; // report 6 times a second.
```

Comment:
Only true for 60 hz...

Reply:
replaced with DesyncDetectionMode enum interval